TESTS/dev_mgmt/connect/main.cpp

```diff
@@ -135,51 +133,20 @@ void spdmc_testsuite_connect(void) {
 
     test_case_finish("Connect to " TEST_NETWORK_TYPE, iteration + (net_status == 0), (net_status != 0));
 
-    test_case_start("Initialize " TEST_BLOCK_DEVICE_TYPE "+" TEST_FILESYSTEM_TYPE, 2);
+    test_case_start("Initialize Storage", 2);
+
     logger("[INFO] Attempting to initialize storage.\r\n");
 
     // Default storage definition.
     BlockDevice* bd = BlockDevice::get_default_instance();
-    SlicingBlockDevice sd(bd, 0, MBED_CONF_APP_BASICS_TEST_FS_SIZE);
-#if TEST_USE_FILESYSTEM == FAT
-    FATFileSystem fs("fs", &sd);
-#else
-    LittleFileSystem fs("fs", &sd);
-#endif
-
-    test_case_finish("Initialize " TEST_BLOCK_DEVICE_TYPE "+" TEST_FILESYSTEM_TYPE, iteration + 1, 0);
-
-    if (iteration == 0) {
-        test_case_start("Format " TEST_FILESYSTEM_TYPE, 3);
-        logger("[INFO] Resetting storage to a clean state for test.\n");
-
-        int storage_status = fs.reformat(&sd);
-        if (storage_status != 0) {
-            storage_status = sd.erase(0, sd.size());
-            if (storage_status == 0) {
-                storage_status = fs.format(&sd);
-                if (storage_status != 0) {
-                    logger("[ERROR] Filesystem init failed\n");
-                }
-            }
-        }
-
-        // Report status to console.
-        if (storage_status == 0) {
-            logger("[INFO] Storage format successful.\r\n");
-        } else {
-            logger("[ERROR] Storage format failed.\r\n");
-            test_failed();
-        }
-
-        test_case_finish("Format " TEST_FILESYSTEM_TYPE, (storage_status == 0), (storage_status != 0));
-    }
+    FileSystem *fs = FileSystem::get_default_instance();
```

Comment:
`FileSystem::get_default_instance()` is not documented by the Mbed OS architect team and there are ongoing debates about the behavior of this function. Also there are different mount points for different block devices which makes the behavior of the functionality very inconsistent. For now, I'd rather keep the behavior where the test decides what filesystem will be used.

Reply:
I can restore it , if this is necessary
changed it also in main function to use the same way, never try it other than FileSystem::get_default_instance()

I am not sure it will work with partition mode.
Can't put back the format , it won't work now.
but formatting done anyway.